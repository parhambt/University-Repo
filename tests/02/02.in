reserve_table 1 1
reserve_table 12 1
switch 1
show_table_info 1