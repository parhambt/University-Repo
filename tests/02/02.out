TODO: 1
ONGOING: 0
DONE: 0
